### ISSUE TITLE
Error when path is not defined

### ISSUE THREAD

**Issue by pjasiun** [ckeditor/ckeditor5-dev-compiler#26]
If I do not define `MODULE_DIR.amd` or `MODULE_DIR.cjs` and call:
```
gulp compile --formats esnext
```
I got an error:
```
Path must be a string. Received undefined
```

These options should not be needed if I do not use these formats.

**Comment by pjasiun** [comment]
Also if I set `MODULE_DIR.amd` or `MODULE_DIR.cjs` to the empty string...
```
gulp compile --formats esnext
```
removes all files in my projects. Including `node_modules`. Then throw error that it can not find a module.

**Comment by Reinmar** [comment]
Fixed by #43.
